Add "ASUS PRIME TRX40 PRO-S" entry to usbmix_ctl_maps.
https://bugzilla.kernel.org/show_bug.cgi?id=206543

diff -uar a/sound/pci/hda/hda_intel.c b/sound/pci/hda/hda_intel.c
--- a/sound/pci/hda/hda_intel.c	2024-02-06 11:10:32.000000000 -0600
+++ b/sound/pci/hda/hda_intel.c	2024-02-25 02:59:50.925399704 -0600
@@ -2068,7 +2068,7 @@
  * should be ignored from the beginning.
  */
 static const struct pci_device_id driver_denylist[] = {
-	{ PCI_DEVICE_SUB(0x1022, 0x1487, 0x1043, 0x874f) }, /* ASUS ROG Zenith II / Strix */
+	{ PCI_DEVICE_SUB(0x1022, 0x1487, 0x1043, 0x874f) }, /* ASUS ROG Zenith II / Strix / Prime */
 	{ PCI_DEVICE_SUB(0x1022, 0x1487, 0x1462, 0xcb59) }, /* MSI TRX40 Creator */
 	{ PCI_DEVICE_SUB(0x1022, 0x1487, 0x1462, 0xcb60) }, /* MSI TRX40 */
 	{}
diff -uar a/sound/usb/mixer_maps.c b/sound/usb/mixer_maps.c
--- a/sound/usb/mixer_maps.c	2024-02-06 11:10:32.000000000 -0600
+++ b/sound/usb/mixer_maps.c	2024-02-25 02:58:55.537205373 -0600
@@ -645,6 +645,11 @@
 		.map = trx40_mobo_map,
 		.connector_map = trx40_mobo_connector_map,
 	},
+	{	/* ASUS PRIME TRX40 PRO-S */
+		.id = USB_ID(0x0b05, 0x1918),
+		.map = trx40_mobo_map,
+		.connector_map = trx40_mobo_connector_map,
+	},
 	{	/* MSI TRX40 Creator */
 		.id = USB_ID(0x0db0, 0x0d64),
 		.map = trx40_mobo_map,
